File: examples/courseman/msa/modules/service2/academic-service/src/main/java/org/jda/example/coursemanmsa/academic/model/Enrolment.java
package org.jda.example.coursemanmsa.academic.model;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;


@Getter @Setter @ToString
public class Enrolment{

	private int id;
	private String studentId;
	private int coursemoduleId;
	private double internalmark;
	private double exammark;
	private String finalgrade;
	private Coursemodule coursemodule;
	private Student student;

}

File: examples/courseman/fereactnative/src/data_types/Enrolment.ts
import {Student} from './Student';
import {CourseModule} from './CourseModule';

export interface Enrolment {
  id: number;
  student: Omit<Student, 'enrolments'>;
  courseModule: CourseModule;
  internalMark?: number;
  examMark?: number;
  finalGrade?: string;
  finalMark?: number;
}
